lib/katello/repository_types/README.md

```diff
@@ -28,24 +60,45 @@ Katello::RepositoryTypeManager.register(::Katello::Repository::YUM_TYPE) do
 ```
 This declares the the repository type, specifying a type identifier (e.g. 'yum')
 
-### Service class
+### Service class and friends
```

Comment:
lol, cute

It would be nice for the "friends" to be fully documented too in the future, but not necessary to block on it.... maybe throw a card on the backlog to revisit?

Reply:
Yeah the generic content type framework would need a full-blown technical docs effort.